examples/rtic.rs

```diff
@@ -3,7 +3,6 @@
 #![no_std]
 #![no_main]
 
-extern crate panic_itm;
```

Comment:
Every examples needs a `#[panic_handler]` to compile. For the examples that don't use logging, then I'd say `use panic_halt as _;` is fine